code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -271,6 +271,11 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 
 			for(var/M in part.markings)
 				icon_key += "[M][part.markings[M]["color"]]"
+			
+			// VOREStation Edit Start
+			if(part.nail_polish)
+				icon_key += "_[part.nail_polish.icon]_[part.nail_polish.icon_state]_[part.nail_polish.color]"
+			// VOREStation Edit End

```

Comment:
I'm not sure doing [icon] will do something reasonable. ... What does that actually get replaced with? I see they do it up a few lines, but I'm not sure about that either.

Reply:
It converts the file path to a string. I get the feeling it'd be an internal path in the rsc or whatever if it was the result of icon ops rather than a file operation.